tests/default/data/sql-ast-narrowing.php

```diff
@@ -365,4 +365,18 @@ public function joinSelectOutsideCondition(PDO $pdo): void
         assertType('PDOStatement<array{adaid: int<-32768, 32767>, 0: int<-32768, 32767>, eadavk: numeric-string|null, 1: numeric-string|null}>', $stmt);
     }
 
+    public function multipleJoins(PDO $pdo): void
+    {
+        $stmt = $pdo->query('SELECT adaid from ada join ak on adaid = eladaid inner join typemix on adaid = c_int');
```

Comment:
Please also select columns from within the join condition, make sure these are narrowed properly and make sure folumns from joined tables not part of the condition are also narrowed properly.

Reply:
I tried, please check, is that you you had in mind?